index.html

```diff
@@ -275,28 +284,30 @@ <h2 class="individual__title">
                     <span class="individual__pager-link"></span>
                     <span class="individual__pager-link individual__pager-link--last"></span>
                 </div>
-                <div class="individual__pager-left">
+                <div class="slider-button slider-button--left">
```

Comment:
It's still an element of `individual` block, change name to `individual__slider-button`

Reply:
This button used in ```about-us``` section as well, do I need to rename it?